### PR TITLE
Merge llvm 25bbc0dcb3

### PR DESCRIPTION
**Pull request by sjackman**

### CODE REVIEW

File: Formula/llvm.rb [hunk 1]
```diff
@@ -54,80 +53,79 @@ class Llvm < Formula
     end
```
Comment (rwhogg) [inline Formula/llvm.rb]: Do we have to bump the version number on `libcxxabi` too?

Comment (sjackman) [inline Formula/llvm.rb]: Yes. Thanks, Bob. I've updated and merged. the PR without a bottle.

**Comment by sjackman** [comment]
@maxim-belkin `brew test-bot --tap=homebrew/core --keep-old took more than 120 minutes to run`
Did you build this bottle manually on Docker last time?

**Comment by maxim-belkin** [comment]
I don't remember if I was the last one to build it (this might have been you, Shaun) but yes, I was building llvm bottles manually.

**Comment by sjackman** [comment]
Would you like to build a bottle for `llvm` 4.0.1?

**Comment by maxim-belkin** [comment]
on it!

**Pull request closed without merging**